frontend/app-development/features/dataModelling/hooks/useValidateSchemaName.test.ts

```diff
@@ -0,0 +1,120 @@
+import { SCHEMA_NAME_MAX_LENGTH, useValidateSchemaName } from './useValidateSchemaName';
+import { useAppMetadataQuery } from 'app-shared/hooks/queries';
+import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmentParams';
+import type { DataModelMetadata } from 'app-shared/types/DataModelMetadata';
+import { textMock } from '@studio/testing/mocks/i18nMock';
+import { renderHook, act } from '@testing-library/react';
+import {
+  dataModel1NameMock,
+  jsonMetadata1Mock,
+} from '../../../../packages/schema-editor/test/mocks/metadataMocks';
+import type { ApplicationMetadata } from 'app-shared/types/ApplicationMetadata';
+import { mockAppMetadata } from '../../../layout/PageHeader/SubHeader/SettingsModalButton/SettingsModal/mocks/applicationMetadataMock';
+
+//Test data
+const org = 'test-org';
+const app = 'test-app';
+const dataModels: DataModelMetadata[] = [jsonMetadata1Mock];
+const appMetaData: ApplicationMetadata = mockAppMetadata;
+
+jest.mock('app-shared/hooks/queries', () => ({
+  useAppMetadataQuery: jest.fn(),
+}));
```

Comment:
Same comment as earlier; we dont need to mock this. We can put it on the cache and send the queryClient to `renderHookWithProviders`

Reply:
Fixed